CONTRIBUTING.md

```diff
@@ -1,61 +1,91 @@
-This is a repo to design a beta version of 18f.gsa.gov based on the U.S.
-Web Design Standards and with a complete refresh of much of the content.
-
-## Where Contributions Go
-
-Thanks for planning to contribute. Please read the below guidelines to know what to expect before submitting a contribution. Submit contributions to https://github.com/18F/18f.gsa.gov as pull requests to the `master` branch. 18F team members should submit pull requests from a branch. Outside contributors will need to fork this repo.
-
-## Front end architecture
-### CSS
-- Styling will be built from the US Web Design Standards.
-- CSS methodology will be inherited from the WDS, which inherits mostly from the [18f front end guide](https://pages.18f.gov/frontend/css-coding-styleguide/architecture/).
-  - Use [18F modifed BEM naming convention](https://pages.18f.gov/frontend/css-coding-styleguide/naming/)
-  - Componentized CSS: start with tag rules and only becomes more specific as necessary, using component classes.
-- Will update the WDS library when it publishes a change required by the site. Otherwise will update bi-monthly (see #1877).
-- The codebase will be visual regression tested when a suitable tool is found for 18F.
-- The Sass code should be linted with `scss-lint` (see #1878)
-  - The [18F CSS linting configuration](https://raw.githubusercontent.com/18F/frontend/18f-pages-staging/.scss-lint.yml) will be used.
-  - If linting fails, it will also fail the tests, but not the build.
-- Will default to [semantic HTML5](http://www.w3schools.com/html/html5_semantic_elements.asp).
-
-
-### Images
-- Will use `<svg>` and `xlink` (looking for link) for icons.
-- All blog images should be under 800kb in total, un-minified size.
+This is a repo to design a beta version of 18f.gsa.gov using the [U.S. Web Design Standards](https://standards.usa.gov/) and to completely refresh most of the content.
```

Comment:
Let's clarify this to indicate this repo is for 18f.gsa.gov, not a beta version. This is my bad, should have caught that before relaunching the site.